Authenticator/Source/TokenListViewController.swift

```diff
@@ -182,17 +197,24 @@ extension TokenListViewController {
 extension TokenListViewController {
     func updateWithViewModel(viewModel: TokenList.ViewModel) {
         let changes = changesFrom(self.viewModel.rowModels, to: viewModel.rowModels)
+        let filtering = viewModel.isFiltering || self.viewModel.isFiltering
         self.viewModel = viewModel
-        updateTableViewWithChanges(changes)
+        updateTableViewWithChanges(changes, whileFiltering: filtering )
         updatePeripheralViews()
     }
 
-    private func updateTableViewWithChanges(changes: [Change]) {
+    private func updateTableViewWithChanges(changes: [Change],
+                                            whileFiltering isFiltering: Bool = false) {
         // TODO: Scroll to a newly added token (added at the bottom)
         if changes.isEmpty || preventTableViewAnimations {
             return
         }
 
+        if isFiltering {
+            tableView.reloadData()
```

Comment:
Yeah, it was unnecessary. When searching I wanted to see my results _right away_.